test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -154,7 +115,7 @@ public void GenerateStartTag_ReturnsNothingIfWhitespaceTagName()
             var output = tagHelperOutput.GenerateStartTag();
 
             // Assert
-            Assert.Empty(output);
+            Assert.Null(output);
```

Comment:
I'd rather it return `null` (if supress output is called or tagname is null) then instantiating a new `TagHelperContent` for no reason.